src/main/java/com/northconcepts/datapipeline/foundations/examples/schema/GenerateJavaCodeFromSchema.java

```diff
@@ -0,0 +1,82 @@
+package com.northconcepts.datapipeline.foundations.examples.schema;
+
+import com.northconcepts.datapipeline.core.FieldType;
+import com.northconcepts.datapipeline.foundations.core.Attributes;
+import com.northconcepts.datapipeline.foundations.core.Tags;
+import com.northconcepts.datapipeline.foundations.schema.SchemaDef;
+import com.northconcepts.datapipeline.foundations.schema.EntityDef;
+import com.northconcepts.datapipeline.foundations.schema.NumericFieldDef;
+import com.northconcepts.datapipeline.foundations.schema.TextFieldDef;
+import com.northconcepts.datapipeline.foundations.schema.IndexDef;
+import com.northconcepts.datapipeline.foundations.schema.EntityRelationshipDef;
+import com.northconcepts.datapipeline.foundations.schema.RelationshipCardinality;
+import com.northconcepts.datapipeline.foundations.schema.ForeignKeyAction;
+import com.northconcepts.datapipeline.foundations.schema.PrimaryKeyType;
+import com.northconcepts.datapipeline.foundations.sourcecode.CodeWriter;
+import com.northconcepts.datapipeline.foundations.sourcecode.JavaCodeBuilder;
+
+public class GenerateJavaCodeFromSchema {
+
+    public static void main(String[] args) {
+
+        JavaCodeBuilder code = new JavaCodeBuilder();
+        CodeWriter codeWriter = code.getSourceWriter();
+        codeWriter.println("public class GenerateJavaCodeFromSchema {");
```

Comment:
This is too [Inception](https://www.imdb.com/title/tt1375666/), let's call it `DataPipelineExample` if you have no concerns.

Reply:
Done